scripts/install-for-ci.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
-fetch
 set -e
 
+if [ "x$TRAVIS" = "xtrue" -a "x$ROSWELL_INSTALL_DIR" = "x" ]; then
+    ROSWELL_INSTALL_DIR=$HOME/usr
+    PATH=$HOME/usr/bin:$PATH
```

Comment:
These variables will set only while in this script, so `ros` cannot be found outside.
https://travis-ci.org/fukamachi/quri/jobs/158439270#L523-L534